content/docs/user-guide/packaging/project-export/project-export-android.md

```diff
@@ -0,0 +1,117 @@
+export STOREPASS=o3depass
+export ALIAS=o3dekey
+export KEYPASS=o3depass
+export KEYSIZE=2048
+export VALIDITY=10000
+export DNAME="cn=o3de-android-project, ou=o3de, o=LF, c=US"
+keytool -genkey -keystore $KEYSTORE_FILE_PATH -storepass $STOREPASS  -alias $ALIAS -keypass $KEYPASS -keyalg RSA -keysize $KEYSIZE -validity $VALIDITY -dname $DNAME
+```
+5. Make sure the O3DE bootstrap registry is enabled for Android. To do so, go to your O3DE installation and edit the file at `$O3DE_ENGINE_PATH/Registry/bootstrap.setreg` so that the `assets` field uses `android`. Like so:
+```
+"assets": "android",
+```
+6. Edit the `$O3DE_ENGINE_PATH/Registry/AssetProcessorPlatformConfig.setreg` file at line 67 by uncommenting the line:
+```
+//"android": "enabled", -> "android": "enabled",
+```
+This will tell the asset processor to make sure Android assets are cached when building the project.
+
+7. Make sure your project has the right configuration settings for Android. You can use the `android-configure` CLI tool to ensure this is setup. To see what is currently configured, run:
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure -l
+```
+You can also validate your current configuration using:
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --validate
+```
+If you do not already have Android configured, or encounter issues with the validate command, you can run the following:
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value platform.sdk.api=30
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value ndk.version=25.*
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value android.gradle.plugin=8.1.0
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value sdk.root=/path/to/android/sdk
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value signconfig.store.file=/path/to/android-key.keystore
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value signconfig.key.alias=o3dekey
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value asset.mode=PAK
+```
+Note: you must already know where sdk.root and signconfig.store.file exist on your hard drive, and use those paths accordingly. To setup the keypass and storepass, you must use the following commands
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-password signconfig.store.password
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-password signconfig.key.password
+```
+Make sure the passwords supplied here match what you used in the keystore file, and that you remember what they are.
+
+## Quickstart
+### Running the Export Script
+For building in release mode with bundled PAK files for your APK, you can use the following export command:
+```
+export O3DE_ENGINE_PATH=/path/to/o3de
+export PROJECT_PATH=/path/to/project
+cd $PROJECT_PATH
+$O3DE_ENGINE_PATH/scripts/o3de.sh export-project -es export_source_android.py -pp . -abp build/game-android -ll INFO --config release --asset-mode PAK
```

Comment:
The example above uses `.bat` which make this page appear windows specific. But this line switches to `o3de.sh` which is mac/linux specific. Suggest you use windows paths for the entire doc for consistency, especially if windows was the platform you tested these steps with.